hotpdf/processor.py

```diff
@@ -52,13 +61,19 @@ def __call_ghostscript(
     if last_page:
         command_line_args.append(f"-dLastPage={last_page}")
 
-    command_line_args.extend([f'-sOutputFile="{temp_xml_file_name}"', f'"{file_path}"'])
+    command_line_args.append(f'-sOutputFile="{temp_xml_file_path}"')
+
+    # Uses gs in pipe mode
+    pipe = type(source) is bytes
+    if pipe:
+        command_line_args.append("-")
+    else:
+        command_line_args.append(str(source))
```

Comment:
make it a cool oneliner 😎 

`command_line_args.append("-" if pipe else (str(source))
`

Reply:
sure ahah
wanted to keep it readable ahaha

Reply:
Done :)